src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -130,6 +131,17 @@ const uploadUrl = `${
   environment.API_URL
 }/simple_forms_api/v1/simple_forms/submit_supporting_documents`;
 
+const fileWithTextConfigBoilerPlate = {
+  fileTypes,
+  fileUploadUrl: uploadUrl,
+  attachmentSchema: {
+    'ui:title': 'Document type',
+  },
+  attachmentName: {
+    'ui:title': 'Document name',
+  },
+};
+
```

Comment:
This was present in every file upload config, so pulled it out here to reuse/condense. At some point in the future will probably look at refactoring the overall fileUploadUI usage, but this tidies things up for now.